app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -85,6 +97,12 @@ class ApiConnector @Inject() (
 
 object ApiConnector {
 
+  sealed abstract class SID(val value: String)
```

Comment:
Perhaps we should have the `ResponseBuilder` use these?

Reply:
That sounds like a good idea, I'll make that change now.

Reply:
Confirmed.